sharktank/sharktank/ops/signatures.py

```diff
@@ -1098,6 +1138,34 @@ def _scatter__trampoline(
         d.fail(tensors)
 
 
+@overridable
+def scatter_add(
+    input: AnyTensor, dim: int, index: AnyTensor, src: AnyTensor | Number
+) -> AnyTensor:
+    """
+    See torch.scatter_add
+    NOTE: Does not modify the inout tensor in place for ShardedTensors, will return copy.
```

Comment:
`scatter_add` shouldn't modify any of them in-place. Only `scatter_add_` should be in-place.

Reply:
Sorry for that copy-paste remnant. Its removed.